content/applications/sales/sales/products_prices/returns/returns_refunds.rst

```diff
@@ -0,0 +1,104 @@
+item without having to zero out the quantity.
+
+.. image:: returns_refunds/Reverse-transfer-popup.png
+  :align: center
+  :alt: Reverse Transfer Pop-Up
+
+Clicking **Return** generates a new Warehouse Operation for the incoming returned product(s). After
+the receiving the return, the team can validate the Warehouse Operation. Head back to the original
+Sales Order, and the Delivered Quantity reflects the difference between the original and the
+returned quantities.
+
+.. image:: returns_refunds/updated-sales-quantities.png
+  :align: center
+  :alt: Updated Sales Order Quantities
+
+The customer receives an invoice only for the products they're keeping.
+
+Case 2: Returning products after invoicing
+==========================================
+Sometimes, customers receive and/or pay their invoice before deciding to return an item within
+your company’s return window. In these cases, a return using only Reverse Transfers is impossible
+since validated or sent invoice cannot be changed, but Reverse Transfers can be used in
+conjunction with Credit Notes to complete the customer's return.
+
+Navigate to the customer’s Sales Order. If a Sales Order has any registered payment on it, the
+payment details are located in the chatter, and the invoice itself has a **Paid** banner across it.
+
+A customer wants to return 10 of the 20 products that they purchased, already
+paid for, and received. Starting a return in this case begins the same as Case 1 by clicking on
+the validated Delivery Order, then on  **Return** to open the Reverse Transfer pop-up. Edit the
+quantity from 20 to 10 since the customer only wants to return half of the order, and click
+**Return**. Just as in Case 1, this action generates a new Warehouse Operation for the incoming
+returned product, which can then be validated once received. Back on the Sales Order, if **Update
+Quantities** was selected, the return product's line item now reflects the change in amount under
+delivered quantities.
+
+.. image:: returns_refunds/case-2-updated-sales-quantities.png
+  :align: center
+  :alt: Updated Delivered Quantities
+
+Since the items have already been paid for, the validated invoice needs to be changed to reflect
+that your team processed a return. Tap on the **i** icon next to the paid line item, and the
+**Payment Info** pop-up appears and provides the option to unreconcile this payment. Click on
+**Unreconcile**.
+
+.. image:: returns_refunds/unreconcile-button.png
+  :align: center
+  :alt: Unreconcile Button
+
+After the invoice is unreconciled, the options for **Send & Print** and **Register Payment** become
+available again alongside a note that there are outstanding payments for the customer.
+
+.. note::
+  The Accounting application needs to be installed for the **Unreconcile** option to be available
+  on an invoice.
+
+Now, select **Add Credit Note**, and a pop-up menu appears where several selections need to be
+made.
```

Comment:
```suggestion
Now, select **Add Credit Note**, and choose (...)
```
I would straight describe what needs to be done, the second part of the sentence is not really necessary.